packages/sanity/src/core/form/inputs/files/FileInput/FileInput.tsx

```diff
@@ -303,17 +297,16 @@ export class BaseFileInput extends React.PureComponent<BaseFileInputProps, BaseF
     this.setState({selectedAssetSource: null})
   }
 
-  hasFileTargetFocus() {
-    return this.props.focusPath?.[0] === 'asset'
-  }
-
-  handleFileTargetFocus = (event: FIXME) => {
+  handleFileTargetFocus = (event: React.FocusEvent) => {
     // We want to handle focus when the file target element *itself* receives
     // focus, not when an interactive child element receives focus. Since React has decided
     // to let focus bubble, so this workaround is needed
     // Background: https://github.com/facebook/react/issues/6410#issuecomment-671915381
```

Comment:
👋 does this comment still hold given the changes?

Reply:
It does! We still need this check to catch focus events triggered by interactive child elements